db/migrate/20150717164408_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :username, null: false
+      t.string :token, null: false
+      t.string :email
```

Comment:
This is new, I think we don't want to email to be nil either, since it will be the mean for us to send notifications to the users.